addons/mail/static/src/composer/composer.xml

```diff
@@ -76,11 +76,11 @@
                             <button class="btn border-0 fa fa-smile-o rounded-pill" aria-label="Emojis" t-on-click="onClickAddEmoji" t-ref="emoji-picker"/>
                             <FileUploader t-if="allowUpload" multiUpload="true" onUploaded="(data) => { attachmentUploader.uploadData(data) }">
                                 <t t-set-slot="toggler">
-                                    <button t-att-disabled="!state.active" class="btn fa fa-paperclip border-0 rounded-pill" title="Attach files" aria-label="Attach files" type="button" t-on-click="onClickAddAttachment"/>
+                                    <button t-att-disabled="!state.active" class="o-mail-Composer-attachFiles btn fa fa-paperclip border-0 rounded-pill" title="Attach files" aria-label="Attach files" type="button" t-on-click="onClickAddAttachment"/>
```

Comment:
Maybe "o-mail-Composer-action-attachFiles" and "o-mail-Composer-action-fullComposer" as those are sub-actions of the "o-mail-Composer-actions" node?

Reply:
We follow the BEM nomenclature `o-moduleName-ComponentName-partName`, so keeping just `attachFiles` is good.
If the "action" part is useful e.g. to style all actions in CSS, better make another classname e.g. `o-mail-Composer-action`. But in general the less classnames the better :)